BlackMarket/Views/Settings/Settings.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  Settings.swift
+//  BlackMarket
+//
+//  Created by Santiago Andracchi on 26/09/2023.
+//
+
+import SwiftUI
+
+struct Settings: View {
```

Comment:
Please remember it is a good practice to add a prefix to provide more context amongst other benefits.
I use to suffix full screens with `Screen` and subviews with `View`. i.e:
```
struct DashboardScreen: View {
 var body... {
   ProductListView()
 }
}
```